iree/compiler/Codegen/LLVMGPU/Verifiers.cpp

```diff
@@ -191,9 +191,6 @@ LogicalResult verifyGPUMatmulTensorCorePipeline(
     bytesSize = type.getElementType().getIntOrFloatBitWidth() / 8;
   } else if (RankedTensorType type = inputType.dyn_cast<RankedTensorType>()) {
     bytesSize = type.getElementType().getIntOrFloatBitWidth() / 8;
-  } else if (UnrankedTensorType type =
```

Comment:
actually you can do something even simpler you can do:
`inputType.cast<ShapedType>().getElementType();`
Matmul operand has to be shaped type so it is fine to do the cast directly.

Reply:
This way you don't need separate case for memref and tensor.

Reply:
also please update the the simt verifier as well.